packages/marqus-desktop/src/renderer/components/Editor.tsx

```diff
@@ -126,6 +133,22 @@ const setContent: Listener<"editor.setContent"> = async ({ value }, ctx) => {
   await debouncedInvoker("notes.update", noteId, { content });
 };
 
+export const setModelViewState: Listener<"editor.setModelViewState"> = (
```

Comment:
Moving this should have been done when we moved `setContent` so this is basically just cleanup.